index.src.html

```diff
@@ -36,6 +36,7 @@ <h1>Self-Review Questionnaire: Security and Privacy</h1>
 
 Version History: https://github.com/w3ctag/security-questionnaire/commits/master/index.src.html
 !Bug Reports: <a href="https://github.com/w3ctag/security-questionnaire/issues/new">via the w3ctag/security-questionnaire repository on GitHub</a>
+Repository: https://github.com/w3ctag/security-questionnaire/
```

Comment:
This can be just
```suggestion
Repository: w3ctag/security-questionnaire
```
Per https://tabatkins.github.io/bikeshed/#metadata-repository.